README.md

```diff
@@ -1,4 +1,681 @@
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
-Coming soon!
+# Eventstreams_Go_SDK 0.0.1
+Go client library to interact with the various [IBM Cloud Eventstreams APIs](https://cloud.ibm.com/apidocs?category=eventstreams_sdkv1).
+
+Disclaimer: this SDK is being released initially as a **pre-release** version.
+Changes might occur which impact applications that use this SDK.
+
+## Table of Contents
+<!--
+  The TOC below is generated using the `markdown-toc` node package.
+
+      https://github.com/jonschlinkert/markdown-toc
+
+  You should regenerate the TOC after making changes to this file.
+
+      npx markdown-toc -i README.md
+  -->
+
+<!-- toc -->
+
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+    + [`go get` command](#go-get-command)
+    + [Go modules](#go-modules)
+    + [`dep` dependency manager](#dep-dependency-manager)
+- [Using the SDK](#using-the-sdk)
+- [Questions](#questions)
+- [Issues](#issues)
+- [Open source @ IBM](#open-source--ibm)
+- [Contributing](#contributing)
+- [License](#license)
+
+<!-- tocstop -->
+
+## Overview
+
+The IBM Cloud Eventstreams Go SDK allows developers to programmatically interact with the following IBM Cloud services:
+
+Service Name | Package name 
+--- | --- 
+<!-- [Example Service](https://cloud.ibm.com/apidocs/example-service) | exampleservicev1 -->
+
+## Prerequisites
+
+[ibm-cloud-onboarding]: https://cloud.ibm.com/registration
+
+* An [IBM Cloud][ibm-cloud-onboarding] account.
+* An IAM API key to allow the SDK to access your account. Create one [here](https://cloud.ibm.com/iam/apikeys).
+* Go version 1.14 or above.
+
+## Installation
+The current version of this SDK: 0.0.1
+
+There are a few different ways to download and install the Eventstreams Go SDK project for use by your
+Go application:
+
+#### `go get` command  
+Use this command to download and install the SDK to allow your Go application to
+use it:
+
+```
+go get -u github.com/IBM/adminrest-go-sdk
+```
+
+#### Go modules  
+If your application is using Go modules, you can add a suitable import to your
+Go application, like this:
+
+```go
+import (
+	"github.com/IBM/adminrest-go-sdk/exampleservicev1"
+)
+```
+
+then run `go mod tidy` to download and install the new dependency and update your Go application's
+`go.mod` file.
+
+#### `dep` dependency manager  
+If your application is using the `dep` dependency management tool, you can add a dependency
+to your `Gopkg.toml` file.  Here is an example:
+
+```
+[[constraint]]
+  name = "github.com/IBM/adminrest-go-sdk"
+  version = "0.0.1"
+
+```
+
+then run `dep ensure`.
+
+## Using the SDK
+For general SDK usage information, please see [this link](https://github.com/IBM/ibm-cloud-sdk-common/blob/main/README.md)
+
+## Questions
```

Comment:
These sections (till License) could be moved to the bottom of the README

Reply:
No because they are two different files that come from different places. The beginning of the README comes from the golang SDK template which has all the intro stuff and then we add our standard admin rest README to the bottom.  It is dictated by the SDK folks that say LICENCE .  CONTRIBUTION Etc is always at the TOP.